plugins/hotkeys.cpp

```diff
@@ -38,14 +37,13 @@ static bool can_invoke(const string &cmdline, df::viewscreen *screen) {
 }
 
 static int cleanupHotkeys(lua_State *) {
-    DEBUG(log).print("cleaning up old stub keybindings for %s\n", current_focus.c_str());
+    DEBUG(log).print("cleaning up old stub keybindings for:\n %s\n", join_strings(", ", Gui::getCurFocus(true)).c_str());
```

Comment:
can we keep this debug log line internal-newline-free?

Reply:
Oops, forgot to get this newline when I changed the separator earlier, will fix